### DISCUSSION TITLE
Rad Pro: Changing the Roboto Font

### DISCUSSION THREAD

**Discussion by IWasBorg** [Gissio/radpro#164] [Q&A]
I want to know about how I can change the default Roboto font by replacing the Roboto ttf (in ./fonts/) prior to converting with ./fonts/fontconv.exe, and then, how do I package/compile the entire thing into a flash-able firmware for, say, a FNIRSI GC-01 device?

**Comment by Gissio** [comment]
Please [RTFM](https://en.wikipedia.org/wiki/RTFM)!

https://github.com/Gissio/radpro/blob/main/docs/developers.md
